projects/aggregate/.env.development

```diff
@@ -20,4 +20,16 @@ NEXT_PUBLIC_SOURCE_ID=
 NEXT_PUBLIC_META_TWITTER_USERNAME=
 NEXT_PUBLIC_FONT_URLS=
 NEXT_PUBLIC_SOURCE_ICON=
-NEXT_PUBLIC_SOURCE_NAME=
\ No newline at end of file
+NEXT_PUBLIC_SOURCE_NAME=
+
+NEXT_PUBLIC_BACKEND_URL=https://hyperdrive.myddns.me
+NEXT_PUBLIC_BACKEND_TOKEN=cc919b6a43b0921fc30fd3a6dbfdd9cdd636ffa9
```

Comment:
we should re-generate this token just before go-live and spread it around internally, just in case

Reply:
its a dev environment, when go to testnet and mainnet, i will generate new one for each environment